### PR TITLE
feat(GRA-886): update docs for ui

### PR DESCRIPTION
**Pull request by Aceix**

### CODE REVIEW

File: ui-reference.rst [hunk 1]
```diff
@@ -90,27 +90,124 @@ Edit
 (14) **Default Saveconfig:** Typically ignore this. Sets the SaveConfig field on wireguard config.
 (15) **UDP Hole Punching:** Whether or not UDP Hole Punching is turned on (see Network Create notes). Only effects new nodes. Enables or disables "UDP Hole Punching" on the network. When on, clients will first reach out to the server. The server will keep track of public addresses / ports and send these to all other clients in the network. This increases NAT traversibility, but can also cause issues depending on the server environment (if server is in a private network, for example). Typically good to enable if clients will "roam" frequently or are user devices. Typically better to disable if most clients will be servers with well-defined endpoints / ports. If enabled, you can also disable UDP Hole Punching on any individual machine via the UI (see Node section) but it will be enabled by default.
 
+
+Hosts
+======
+
+In simple terms, a host is a netclient or a machine running the netclient software. Netmaker UI allows an admin to conviniently view and configure some host settings remotely.
```
Comment (mattkasun) [inline ui-reference.rst:97]: Computer or machine

File: ui-reference.rst [hunk 2]
```diff
@@ -90,27 +90,124 @@ Edit
...
+
+(1) **ID:** Unique identifier for the host
+(2) **Name:** Name of the host. Defaults to the machine's name.
+(3) **Version:** Version of netclient the host is running.
+(4) **Operating System:** Operating system (OS) the machine is running.
+(5) **Public Key:** Public key of the host. distributed to other hosts.
+(6) **MTU:** Maximum Transmission Unit (MTU) of the host
+(7) **Listen Port:** The wiregaurd listen port.
+(8) **Proxy Listen Port:** The netclient proxy listen port. this is used if `Proxy Enabled` is set to `true`.
+(9) **Verbosity:** Log verbosity (ranges from 1-4). Indicates detailed the host (netclient) will output logs.
```
Comment (mattkasun) [inline ui-reference.rst:137]: Indicates level of detail the host (netclient) will output to logs.

**Review by mattkasun — CHANGES_REQUESTED** [review]

File: ui-reference.rst (hunk 2 above)
Comment (Aceix) [reply ui-reference.rst:137]: rectified

File: ui-reference.rst (hunk 1 above)
Comment (Aceix) [reply ui-reference.rst:97]: rectified

**Review by mattkasun — APPROVED** [review]

**Pull request merged**